setup.py

```diff
@@ -1,34 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
 from setuptools import setup, find_packages
 
-requires = [
-    'numpy',
-    'argparse',
-    'nose'
-]
-
-def readme():
-    with open('README.rst') as f:
-        return f.read()
-
-#if sys.version_info < (3, 2):
-#    requires.append('futures==2.2')
-
-setup(name='graphqa',
-    version='0.1',
-    description='graphqa is a question answering (QA) system built ove
-    knowledge graphs (KG).',
-    url='https://github.com/sdliuyuzhi/graphqa',
-    author='Yuzhi Liu',
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = ['Click>=6.0', ]
+
+setup_requirements = ['pytest-runner', ]
+
+test_requirements = ['pytest', ]
+
+setup(
+    author="Yuzhi Liu",
     author_email='liuyuzhi83@gmail.com',
-    license='BSD 2-Clause License',
-    packages=find_packages(exclude=['tests']),
-    include_package_data=True,
-    test_suite='nose.collector',
-    install_requires=requires,
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Natural Language :: English',
+        "Programming Language :: Python :: 2",
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```

Comment:
I noticed you are using `f` style formatting (e.g. `[f"<{c}> " for c in t]`), which requires the Python version >= 3.6